lib/puppet/provider/foreman_config_entry/cli.rb

```diff
@@ -65,6 +65,8 @@ def value
 
   def value=(value)
     return if resource[:dry]
+
+    value = value.unwrap if value.respond_to?('unwrap')
```

Comment:
I always thought `respond_to` needed a symbol. However, the docs say:
> When the method name parameter is given as a string, the string is converted to a symbol.

That said, I'd still prefer to see a symbol to avoid the conversion:
```suggestion
    value = value.unwrap if value.respond_to?(:unwrap)
```

Reply:
Changed.